README.md

```diff
@@ -7,8 +7,9 @@
   * [About the code](#about-the-code)
   * [Pointers](#pointers)
 * [Installation](#installation)
-  * [System-wide](#system-wide)
-  * [Virtualenv](#virtualenv)
+  * [System-wide with pip](#system-wide-with-pip)
+  * [System-wide from source](#system-wide-from-source)
+  * [virtualenv](#virtualenv)
```

Comment:
I guess this is just an example of title case, where the words in headings follow some special rules for capitalizing some words. We are not doing this here and all other commands below are starting with a small letter, e.g. `hocr-pdf`. I don't want to change all of them.